### DISCUSSION TITLE
Planning for Release 3.19

### DISCUSSION THREAD

**Discussion by CharliePoole** [nunit/nunit-console#1524] [Team Discussions]
I'm working to wrap up 3.19 for release. You can see what's included in the [release milestone](https://github.com/nunit/nunit-console/milestone/32). One of the outstanding items in resolved by a PR, which is waiting for review. The other is awaiting comment to decide what we should actually do about it.

Is there any other work in process that should be included? My hope is to start focusing on V4 after this release.

**Comment by mikkelbu** [comment]
I've not been following the work in the console/engine that closely, so I don't have any work that needs to be included

**Comment by CharliePoole** [comment]
Release 3.19.0 is now out.
